webpack.config.js

```diff
@@ -315,7 +280,7 @@ function generateWebpack(options) {
   if (!options.bundle || options.isApp) {
     // extract the included css file to own file
     const p = new ExtractTextPlugin({
-      filename: (options.isApp || options.moduleBundle ? '[name]' : pkg.name) + (options.min && !options.nosuffix ? '.min' : '') + '.css',
```

Comment:
[name] seems to be the newer version

Reply:
See also https://github.com/phovea/generator-phovea/issues/257